### DISCUSSION TITLE
mergerfs freezes applications at startup if a drive doesn't exist

### DISCUSSION THREAD

**Discussion by ghost** [trapexit/mergerfs#1069] [Q&A] [answered]
I've added the following to my `/etc/fstab`
```
/mnt/T:/mnt/V:/mnt/W:/mnt/X:/mnt/Y:/mnt/Z:/srv/nfs/lenovo/hdd1  /mnt/storage      fuse.mergerfs  allow_other,use_ino,cache.files=off,dropcacheonclose=true,ignorepponrename=true,func.mkdir=epall,x-gvfs-show   0       0
```
Where I've shared the `mnt` folder of another computer through nfs as shown [here](https://forum.manjaro.org/t/error-sharing-nfs-drive/121851/9). If that computer is powered off then when I power up this computer, it freezes when I do anything that uses the storage folder like open a file manager or changing to that folder in the terminal.

Mergerfs is supposed to work even if a disk isn't connected so maybe I've configured something incorrectly?

**Comment by trapexit** [comment]
What happens if you go to the mount points for the NFS share?

**Comment by trapexit** [comment, marked as answer]
If the problem is NFS then mergerfs has no option but to block. mergerfs is a regular app like any other. It can't work around what the kernel does to it. You should ideally be setting up the NFS share to manage these situations better.

https://unix.stackexchange.com/questions/31979/stop-broken-nfs-mounts-from-locking-a-directory
